### PR TITLE
[stable/pgadmin] upgrade to 4.17 and more

### PR DESCRIPTION
**Pull request by rowanruseler**
4.17:
  with 4.16 and up `USER: pgadmin` got added to the Dockerfile, this
  caused an ACL issue. securityContext and initContainers is added to
  resolve this issue.

other:
  - fixed dynamic PV provisioning
  - livenessProbe
  - readinessProbe
  - added more comments in values.yaml
  - updated README.md with new configuration items
  - chart bump version

Signed-off-by: Rowan Ruseler <rowanruseler@gmail.com>

### CODE REVIEW

File: stable/pgadmin/templates/deployment.yaml [hunk 1]
```diff
@@ -51,24 +61,32 @@ spec:
...
+    {{- if .Values.securityContext }}
+      securityContext:
+        {{- .Values.securityContext | toYaml | nindent 8 }}
+    {{- end }}
+    {{- if .Values.affinity }}
       affinity:
-        {{- toYaml . | nindent 8 }}
+        {{- .Values.affinity | toYaml | nindent 8 }}
     {{- end }}
-    {{- with .Values.tolerations }}
```
Comment (davidkarlsen) [inline stable/pgadmin/templates/deployment.yaml:71]: more of a personal taste - but I favour using `with` as it is easier to read and keep consistent due to no duplication.

File: stable/pgadmin/templates/pvc.yaml [hunk 2]
```diff
@@ -1,19 +1,26 @@
-{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
```
Comment (davidkarlsen) [inline stable/pgadmin/templates/pvc.yaml:1]: See https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md for info on persistence

File: stable/pgadmin/templates/deployment.yaml (hunk 1 above)
Comment (rowanruseler) [reply stable/pgadmin/templates/deployment.yaml:71]: Would you like me to adjust this? Personally I found this statement more clear. But I don't mind adjusting it back.

File: stable/pgadmin/templates/pvc.yaml (hunk 2 above)
Comment (rowanruseler) [reply stable/pgadmin/templates/pvc.yaml]: Not really sure what I should be looking at, could you perhaps be more specific?

**Review by prydonius — APPROVED** [review]
/lgtm

**Pull request merged**